packages/roosterjs-editor-plugins/lib/HyperLink/HyperLink.ts

```diff
@@ -20,11 +20,12 @@ export default class HyperLink implements EditorPlugin {
      * @param getTooltipCallback A callback function to get tooltip text for an existing hyperlink.
      * Default value is to return the href itself. If null, there will be no tooltip text.
      * @param target (Optional) Target window name for hyperlink. If null, will use "_blank"
-     * @param linkMatchRules (Optional) Rules for matching hyperlink. If null, will use defaultLinkMatchRules
+     * @param onLinkOpen (Optional) Open link callback
      */
     constructor(
         private getTooltipCallback: (href: string) => string = href => href,
-        private target?: string
+        private target?: string,
+        private onLinkOpen?: (anchor: HTMLAnchorElement) => void
```

Comment:
I can add it back. I figured, let's keep it minimal and add new params as needed. No strong opinion on this.

Reply:
With Firefox, you may need to preventDefault() of the event in you handler since it will just let mouse click open the link, so I think event object is a mandatory one.

Reply:
makes sense